### DISCUSSION TITLE
Apple Lossless Audio Playback on AMD RX 6700 XT with NootRX

### DISCUSSION THREAD

**Discussion by joaodriessen** [ChefKissInc/NootRX#83] [Q&A]
Apologies if this question has been covered before, but I'm trying to figure out how to get Apple Lossless (ALAC) audio playback working on my system.

Previously, I had no issues when using an RX 480 GPU with WhateverGreen, but after upgrading to an AMD RX 6700 XT, I’m unable to play DRM-protected Apple Lossless files. Does NootRX handle this, or is there a specific setup I need to follow?

System specs:

CPU: Intel Skylake 7700k i7
GPU: AMD RX 6700 XT
Audio Interface: RME Fireface UCXII (fully compatible with lossless audio, used for pro audio recording)
macOS Version: Ventura 13.7
OpenCore: 1.0.1

Any help or insights would be greatly appreciated!

**Comment by VisualEhrmanntraut** [comment]
#4
